model/src/main/scala/explore/model/RootModel.scala

```diff
@@ -16,9 +16,11 @@ import monocle.Prism
 import monocle.function.Possible.possible
 import monocle.macros.GenPrism
 import monocle.macros.Lenses
+import monocle.std.option.some
```

Comment:
Are you using this?

Reply:
Not anymore. I'm sending another commit with organized imports.